skimage/_shared/tests/test_interpolation.py

```diff
@@ -20,7 +20,7 @@ def test_coord_map():
     assert_array_equal(reflect, expected_reflect)
 
     constant = [coord_map_py(4, n, 'C') for n in range(-6, 6)]
-    expected_constant = [0, 0, 0, 0, 0, 0, 0, 1, 2, 3, 0, 0]
+    expected_constant = [-6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5]
```

Comment:
This cannot produce valid output. What coordinate should it map to? The result should be a constant, which cannot be achieved by mapping to a memory location...